nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/api/AccessResource.java

```diff
@@ -727,10 +729,12 @@ public Response createAccessToken(
         try {
             // attempt to authenticate
             final AuthenticationResponse authenticationResponse = loginIdentityProvider.authenticate(new LoginCredentials(username, password));
-            long expiration = validateTokenExpiration(authenticationResponse.getExpiration(), authenticationResponse.getIdentity());
+            final String rawIdentity = authenticationResponse.getIdentity();
+            long expiration = validateTokenExpiration(authenticationResponse.getExpiration(), rawIdentity);
```

Comment:
Another instance of the logs (inside `validateTokenExpiration`) reflecting `rawIdentity` vs. `mappedIdentity`.